ja/README.md

```diff
@@ -0,0 +1,135 @@
+# イントロダクション
+
+このフレームワークにより、ソフトウェアエンジニアリングマネージャーは、各ポジションに期待されることや、キャリアラダーにおける次のレベルへの計画の立て方などについて、メンバーと有意義な会話をすることができます。
+
+このフレームワークは、米国のハイテク業界における、おおよそ標準的な役割とレベルを使用していますが、企業によってそれぞれ異なります。
+
+このフレームワークでは、レーダーチャートを多用して、あるポジションのさまざまな視点や期待を視覚的に表現しています。
+
+![Template Chart](../charts/template.png)
+
+# キャリアラダー
+
+本フレームワークは以下4つのラダーから成ります。
+
+* [**開発者**](Developer.md): プログラマーやソフトウェアエンジニアとも呼ばれる役割で、深い技術的専門知識が求められます
+* [**テックリード**](TechLead.md): 開発リーダーとも呼ばれ、システムのオーナーであり、ハンズオン開発、アーキテクチャの知識、プロダクションサポートのユニークなバランスを必要とする役割
+* [**テクニカルプログラムマネージャー**](TechnicalProgramManager.md): 複数のチームにまたがるイニシアティブを調整し、完成に導く役割を担います
+* [**エンジニアリングマネージャー**](EngineeringManager.md): 開発マネージャーとも呼ばれ、チームの一貫したデリバリー、キャリアアップ、幸福度に責任を持つ役割です
+
+[テックリード](TechLead.md)と[エンジニアリングマネージャー](EngineeringManager.md)の違いがよくわからないという方は、[テックリードとエンジニアリングマネージャーの違い](TechLead-EngineeringManager.md) のページで詳しく比較しています。
+
+| レベル | Seniority | [開発者](Developer.md) | [テックリード](TechLead.md) | [テクニカルプログラムマネージャー](TechnicalProgramManager.md) | [エンジニアリングマネージャー](EngineeringManager.md) |
```

Comment:
These titles are very long compared to the English version, and as a result the EM column isn't very visible.
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/8797880/181526388-3ba1f502-6f3c-4025-83ba-d547becce3ce.png">

Maybe we can use an abbreviation instead? Or maybe it is more readable if we add some spaces (I haven't tested this 😅, but I'm guessing it will wrap the text in this case):

```suggestion
| レベル | Seniority | [開発者](Developer.md) | [テック リード](TechLead.md) | [テクニカル プログラム マネージャー](TechnicalProgramManager.md) | [エンジニアリング マネージャー](EngineeringManager.md) |
```